core/spam/vote_spam_policy.go

```diff
@@ -433,3 +434,20 @@ func (vsp *VoteSpamPolicy) PreBlockAccept(tx abci.Tx) (bool, error) {
 
 	return true, nil
 }
+
+func (vsp *VoteSpamPolicy) GetStats(partyID string) Statistic {
+	vsp.lock.RLock()
+	defer vsp.lock.RUnlock()
+
+	stats := Statistic{}
+
+	bStats, ok := vsp.partyBlockRejects[partyID]
+	if !ok {
```

Comment:
Minor: this is a little inconsistent with the other spam-stats categories in that for the others, if you pass in a party that hasn't been seen you get counts of `"0"` but for votes you get `""`:
```
{
  "statistics": {
    "proposals": {
      "countForEpoch": "0",
      "maxForEpoch": "3"
    },
    "delegations": {
      "countForEpoch": "0",
      "maxForEpoch": "390"
    },
    "transfers": {
      "countForEpoch": "0",
      "maxForEpoch": "20"
    },
    "nodeAnnouncements": {
      "countForEpoch": "0",
      "maxForEpoch": "1"
    },
    "votes": {
      "countForEpoch": "" <-- this guy
    },
    "pow": {
      "totalTxsInScope": "0"
    }
  }
}
```

Reply:
Thanks Will, I believe this has been addressed now: https://github.com/vegaprotocol/vega/pull/7275/commits/9804a5d4f1f2567c57a6ee0713cb1ea7da56d315